src/doc/src/CHANGELOG.md

```diff
@@ -28,6 +51,28 @@
 
 ### Fixed
 
+- Fixed potential deadlock in `CacheState::lock`
+  [#15698](https://github.com/rust-lang/cargo/pull/15698)
+  [#15699](https://github.com/rust-lang/cargo/pull/15699)
+- Fixed the `--manifest-path` arg being ignored in `cargo fix`
+  [#15633](https://github.com/rust-lang/cargo/pull/15633)
+- When publishing, don't tell people to ctrl-c without knowing consequences.
+  [#15632](https://github.com/rust-lang/cargo/pull/15632)
+- Added missing `--offline` in shell completions.
+  [#15623](https://github.com/rust-lang/cargo/pull/15623)
+- cargo-credential-libsecret: load libsecret only once
+  [#15295](https://github.com/rust-lang/cargo/pull/15295)
+- fix(fingerprint): When failing to find the mtime of a file, report an explicit reason rather than "stale; unknown reason".
+  [#15617](https://github.com/rust-lang/cargo/pull/15617)
```

Comment:
```suggestion
- When failing to find the mtime of a file for rebuild detection, report an explicit reason rather than "stale; unknown reason".
  [#15617](https://github.com/rust-lang/cargo/pull/15617)
```